soccer/NewRefereeModule.cpp

```diff
@@ -283,6 +283,8 @@ void NewRefereeModule::updateGameState(bool blueTeam) {
             break;
         case Command::FORCE_START:
             _state.gameState.state = GameState::Playing;
+            _state.gameState.ourRestart = false;
+            _state.gameState.restart = GameState::None;
```

Comment:
It would be nice to add these to the stopped state as well but I'm not sure if that would break any restarts. If you think it's ok, I can add that + test it out.